perception/multi_object_tracker/include/multi_object_tracker/multi_object_tracker_core.hpp

```diff
@@ -59,6 +61,12 @@ class MultiObjectTracker : public rclcpp::Node
     detected_object_sub_;
   rclcpp::TimerBase::SharedPtr publish_timer_;  // publish timer
 
+  // debug publishers
+  rclcpp::Publisher<autoware_auto_perception_msgs::msg::TrackedObjects>::SharedPtr
+    debug_tentative_objects_pub_;
+  std::unique_ptr<tier4_autoware_utils::StopWatch<std::chrono::milliseconds>> stop_watch_ptr_;
+  std::unique_ptr<tier4_autoware_utils::DebugPublisher> debug_publisher_;
+
```

Comment:
I don't know if the code is for Debug or necessary for logic, so please provide a Debugger class or something to separate the two.

Reply:
OK. All of these code in this PR are for debugs.

Reply:
Debugger process is separated to TrackerDebugger class in https://github.com/autowarefoundation/autoware.universe/pull/5762/commits/14bd9297c281a5d56b6a8a4191a6edab0b5e1504.